ui/src/submissions/index.jsx

```diff
@@ -61,6 +64,16 @@ class Submissions extends Component {
               path={SUBMISSIONS_CONFERENCE}
               component={ConferenceSubmissionPageContainer}
             />
+            <Route
```

Comment:
should we put it under feature flag or only superusers/curators?

Reply:
well we don't have it under `submissions` menu and on homepage, so it's not really discoverable.